books/RayTracingInOneWeekend.html

```diff
@@ -509,8 +528,8 @@
   ![Figure [cam-geom]: Camera geometry](../images/fig-1.03-cam-geom.jpg)
 
 <div class='together'>
-Below in code, the ray `r` goes to approximately the pixel centers (I won’t worry about exactness
-for now because we’ll add antialiasing later):
+Below in code, the ray `r` goes to the current image pixel (I won’t worry about exactness for now
+because we’ll add antialiasing later):
```

Comment:
The rewrite makes things less clear.

In saying "I won't worry about exactness for now" it sounds like we might be accidentally hitting nearby pixels. Which is not the intent of the qualifier, rather that we might not peg the exact pixel center.

Consider reverting, qualifying the exactness comment, or removing the exactness comment

Reply:
I was a bit puzzled about the exactness comment as well. I don't think it's needed; will drop.